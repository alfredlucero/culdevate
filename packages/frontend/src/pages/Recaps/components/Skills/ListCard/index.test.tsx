import React from "react";
import { render } from "@testing-library/react";
import SkillsListCard from "./index";

describe("<SkillsListCard />", () => {
  test("should render without error", () => {
    const { container } = render(<SkillsListCard onClick={() => {}} className="extra-list-class" testId="testId" />);

    expect(container).toMatchSnapshot();
  });
});
