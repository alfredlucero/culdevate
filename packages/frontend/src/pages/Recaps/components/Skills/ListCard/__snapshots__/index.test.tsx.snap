// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SkillsListCard /> should render without error 1`] = `
<div>
  <button
    class="w-64 hover:bg-blue-100 extra-list-class"
    data-testid="testId"
    type="button"
  >
    <div
      class="bg-white shadow-md rounded px-8 py-6 h-56 flex items-center justify-center flex-col border-b-8 border-teal-400 border-solid"
      style="background-color: inherit;"
    >
      <div
        class="mb-6 flex justify-center"
      >
        <div
          class="flex justify-center items-center p-3 rounded-lg bg-gray-200 recap-icon"
        >
          <div
            class="font-sans text-3xl text-teal-400"
            id=""
          >
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-certificate fa-w-16 "
              data-icon="certificate"
              data-prefix="fas"
              focusable="false"
              role="img"
              viewBox="0 0 512 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M458.622 255.92l45.985-45.005c13.708-12.977 7.316-36.039-10.664-40.339l-62.65-15.99 17.661-62.015c4.991-17.838-11.829-34.663-29.661-29.671l-61.994 17.667-15.984-62.671C337.085.197 313.765-6.276 300.99 7.228L256 53.57 211.011 7.229c-12.63-13.351-36.047-7.234-40.325 10.668l-15.984 62.671-61.995-17.667C74.87 57.907 58.056 74.738 63.046 92.572l17.661 62.015-62.65 15.99C.069 174.878-6.31 197.944 7.392 210.915l45.985 45.005-45.985 45.004c-13.708 12.977-7.316 36.039 10.664 40.339l62.65 15.99-17.661 62.015c-4.991 17.838 11.829 34.663 29.661 29.671l61.994-17.667 15.984 62.671c4.439 18.575 27.696 24.018 40.325 10.668L256 458.61l44.989 46.001c12.5 13.488 35.987 7.486 40.325-10.668l15.984-62.671 61.994 17.667c17.836 4.994 34.651-11.837 29.661-29.671l-17.661-62.015 62.65-15.99c17.987-4.302 24.366-27.367 10.664-40.339l-45.984-45.004z"
                fill="currentColor"
              />
            </svg>
          </div>
        </div>
      </div>
      <h4
        class="font-mono font-bold text-gray-800 text-xl text-center"
      >
        Skills
      </h4>
    </div>
  </button>
</div>
`;
